tests/foreman/cli/test_discoveredhost.py

```diff
@@ -104,16 +102,6 @@ def setUpClass(cls):
         # discovered host provisioning.
         cls.configured_env = False
 
-        if bz_bug_is_open(1578290):
-            ssh.command('mkdir /var/lib/tftpboot/boot/fdi-image')
-            ssh.command('ln -s /var/lib/tftpboot/boot/'
-                        'foreman-discovery-image-3.4.4-1.iso-vmlinuz'
-                        ' /var/lib/tftpboot/boot/fdi-image/vmlinuz0')
-            ssh.command('ln -s /var/lib/tftpboot/boot/'
-                        'foreman-discovery-image-3.4.4-1.iso-img'
-                        ' /var/lib/tftpboot/boot/fdi-image/initrd0.img')
-            ssh.command('chown -R foreman-proxy /var/lib/tftpboot/boot/')
-
```

Comment:
One Open Question here @SatelliteQE/quality-engineers,
we are removing all good amount tweaking code in case of issues, similar above case. If tomorrow we get same issue I'm not sure we open a new issue or reopen previously closed issues.

But importatnt thing is that we remove these all lines from code that we again have to reinvent unless someone tries to see history for the quick fix.

Reply:
I think the best would be to keep this code in the commented format instead of removing it. Their is no harm or increase in time either by keeping it commented.

We can keep something like this:
```
'''
Patch for BZ: 1578290
ssh.command('mkdir /var/lib/tftpboot/boot/fdi-image')
ssh.command('ln -s /var/lib/tftpboot/boot/'
         'foreman-discovery-image-3.4.4-1.iso-vmlinuz'
         ' /var/lib/tftpboot/boot/fdi-image/vmlinuz0')
ssh.command('ln -s /var/lib/tftpboot/boot/'
         'foreman-discovery-image-3.4.4-1.iso-img'
         ' /var/lib/tftpboot/boot/fdi-image/initrd0.img')
ssh.command('chown -R foreman-proxy /var/lib/tftpboot/boot/')
'''
```

Reply:
I wouldn't worry about losing workaround if similar issue ever re-surfaces. I would trust component owner to know about this trick. It's probably something that could be added as a note in "how to test component X" page on Mojo, one of these that management asked us to create earlier this year.

And please, don't leave it as a comment. Dead code place is in version control history.

Reply:
+1 to no comment
However, loss of knowledge is a legitimate concern, especially as component ownership will change. Maybe we need to store these kinds of workarounds somewhere. If not in a private comment on the bug itself, then maybe on mojo or a new repo called "deprecated workarounds" or something.

Reply:
Git has a tool for that https://git-scm.com/docs/git-notes
+1 to keep that as a private BZ comment until we figureout better use of git notes.